apps/widget/src/stores/Account.ts

```diff
@@ -47,7 +47,7 @@ export const useAccountStore = defineStore('account', {
                 // silent_redirect_uri: WIDGET_URL + '/signout-popup.html',
                 post_logout_redirect_uri: WIDGET_URL + '/signout-popup.html',
                 popup_post_logout_redirect_uri: WIDGET_URL + '/signout-popup.html',
-                scopes: 'openid account:read erc20:read erc721:read point_balances:read referral_rewards:read point_rewards:read wallets:read wallets:write',
+                scopes: 'openid account:read erc20:read erc721:read point_balances:read referral_rewards:read shopify_rewards:read point_rewards:read wallets:read wallets:write',
```

Comment:
@valeriagrazzini Why is this one requested? Widget doesn't call those resource endpoints right?

Reply:
Ah I see all other resources are also requested. Still I think that might not be necessary since if I remember correctly we read all data through the optionally authenticated `/v1/rewards` endpoint